Research/Research/AnswerTypeObjects.swift

```diff
@@ -469,9 +532,13 @@ public struct AnswerTypeDateTime : RSDBaseAnswerType, Codable, Hashable {
     }
 }
 
-public struct AnswerTypeMeasurement : RSDBaseAnswerType, Codable, Hashable {
+public struct AnswerTypeMeasurement : BaseAnswerType, Codable, Hashable {
+    private enum CodingKeys : String, CodingKey, CaseIterable {
+        case objectType = "type", unit
+    }
     public static let defaultJsonType: JsonType = .number
-    public private(set) var type: AnswerTypeType = .measurement
+    public static let defaultType: AnswerTypeType = .measurement
+    public private(set) var objectType: AnswerTypeType = Self.defaultType
```

Comment:
Capitalized "Self" is a megalomaniac :P

Jokes aside, I didn't know about that use of capitalized self, good to know.